packages/batsignal/batsignal.pacscript

```diff
@@ -1,17 +1,19 @@
 name="batsignal"
-version="1.3.2"
+version="1.5.0"
 url="https://github.com/electrickite/batsignal/archive/refs/tags/${version}.zip"
 build_depends="libnotify-dev make pkg-config gcc"
 replace="batsignal"
 description="A lightweight battery monitor daemon"
-hash="1eb564c94f4a49900196125f02980d747e8eb7901715ad05de1042f7b54d3a20"
+hash="b7ea386a2b14427c70a63b5e91feebc8213618c9d62d82f569c1dfc2f8737854"
 maintainer="Harshwardhan Mehrotra <harshwardhanmehrotra@gmail.com>"
 
 prepare() {
         true
 }
 
 build() {
+  sed -i '40d' Makefile
+  sed -i '41d' Makefile
```

Comment:
What do these do?

Reply:
check the build errors on the first commit they basically remove the hardcoded man location